app/design/adminhtml/Magento/backend/Magento_Ui/web/css/source/module/_data-grid.less

```diff
@@ -429,7 +430,7 @@ body._in-resize {
     .data-grid-actions-cell {
         padding-left: @data-grid-cell__padding-horizontal * 2;
         padding-right: @data-grid-cell__padding-horizontal * 2;
-        position: relative;
+        position: inherit;
         text-align: center;
```

Comment:
any reason of this update position

Reply:
can it's work without it ?

Reply:
hi @adamphh can you explain more about this changes

Reply:
If no specific or special reason i suggest to remove for reduce extra css property

Reply:
@mrtuvn there is no special reason for that, it  can work without position, However I am not sure if it affect to another section.